firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -2311,6 +2311,11 @@ extension BrowserViewController: LegacyTabDelegate {
                     if !loginsForCurrentTab.isEmpty {
                         tab.webView?.accessoryView.reloadViewFor(.login)
                         tab.webView?.reloadInputViews()
+                        TelemetryWrapper.recordEvent(
```

Comment:
@razvanlitianu just to clarify this will always get sent even if the user hasn't tapped on the actual accessory view or selected a login item right?

Reply:
hey @nbhasin2 this will be sent only if the user taps a login field to record that it sees the accessory view. According to @afurlan-firefox this will help differentiate the autofilled metric that was reused from the old automatic autofill.